components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -193,7 +340,15 @@ service Policies {
     option (chef.automate.api.policy).action = "delete";
     option (chef.automate.api.iam.policy).resource = "iam:roles:{id}";
     option (chef.automate.api.iam.policy).action = "iam:roles:delete";
-    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "roles";
+		};
+  };
+  /*
+  Update a role
+
+  Update an existing role. Very similar to create except the ID cannot be changed.
```

Comment:
To be consistent with "Update a policy", delete the first fragment:
```suggestion
  Very similar to create except the ID cannot be changed.
```
And then add:
"When updating a role, supply all of a role's properties, not just the ones you wish to update. Properties that you do not include are reset to empty values."